### PR TITLE
doc:Adding doc for ad777x_iio and nanodac_console

### PR DESCRIPTION
**Pull request by D-Disha**
Added documentation and updated TOC through index.rst

### CODE REVIEW

**Comment by D-Disha** [comment]
Hi all, there are no EVAL board links present for both the projects. Hence, I have not added them. 

There are around 30 generics for nanodac_console. I have double checked the product links, but I'd really appreciate it if anyone could check if all the links are correct and can be accessed.

File: doc/sphinx/source/projects/nanodac_console/nanodac_console.rst [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+`AD5684r <https://www.analog.com/en/products/ad5684r.html>`_ `AD5685r <https://www.analog.com/en/products/ad5685r.html>`_ `AD5686r <https://www.analog.com/en/products/ad5686r.html>`_ 
+`AD5687 <https://www.analog.com/en/products/ad5687.html>`_  `AD5687r <https://www.analog.com/en/products/ad5687r.html>`_  `AD5689r <https://www.analog.com/en/products/ad5689r.html>`_ 
+`AD5697r <https://www.analog.com/en/products/ad5697r.html>`_ `AD5694r <https://www.analog.com/en/products/ad5694r.html>`_ `AD5695r <https://www.analog.com/en/products/ad5695r.html>`_ 
+`AD5696 <https://www.analog.com/en/products/ad5696.html>`_  `AD5696r <https://www.analog.com/en/products/ad5696r.html>`_  `AD5681r <https://www.analog.com/en/products/ad5681r.html>`_ 
+`AD5682r <https://www.analog.com/en/products/ad5682r.html>`_ `AD5683r <https://www.analog.com/en/products/ad5683r.html>`_ `AD5691r <https://www.analog.com/en/products/ad5673r.html>`_ 
+`AD5692r <https://www.analog.com/en/products/ad5692r.html>`_  `AD5693r <https://www.analog.com/en/products/ad5693r.html>`_  `AD5693 <https://www.analog.com/en/products/ad5675r.html>`_
```
Comment (jansunil) [inline doc/sphinx/source/projects/nanodac_console/nanodac_console.rst:19]: The last link says AD5693, but points to AD5675r- Is there a typo here?

File: doc/sphinx/source/projects/nanodac_console/nanodac_console.rst [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+`AD5674 <https://www.analog.com/en/products/ad5674.html>`_  `AD5679r <https://www.analog.com/en/products/ad5679r.html>`_  `AD5686 <https://www.analog.com/en/products/ad5686.html>`_ 
+`AD5684r <https://www.analog.com/en/products/ad5684r.html>`_ `AD5685r <https://www.analog.com/en/products/ad5685r.html>`_ `AD5686r <https://www.analog.com/en/products/ad5686r.html>`_ 
+`AD5687 <https://www.analog.com/en/products/ad5687.html>`_  `AD5687r <https://www.analog.com/en/products/ad5687r.html>`_  `AD5689r <https://www.analog.com/en/products/ad5689r.html>`_ 
+`AD5697r <https://www.analog.com/en/products/ad5697r.html>`_ `AD5694r <https://www.analog.com/en/products/ad5694r.html>`_ `AD5695r <https://www.analog.com/en/products/ad5695r.html>`_ 
+`AD5696 <https://www.analog.com/en/products/ad5696.html>`_  `AD5696r <https://www.analog.com/en/products/ad5696r.html>`_  `AD5681r <https://www.analog.com/en/products/ad5681r.html>`_ 
+`AD5682r <https://www.analog.com/en/products/ad5682r.html>`_ `AD5683r <https://www.analog.com/en/products/ad5683r.html>`_ `AD5691r <https://www.analog.com/en/products/ad5673r.html>`_
```
Comment (jansunil) [inline doc/sphinx/source/projects/nanodac_console/nanodac_console.rst:18]: The last link says AD5691, but points to AD5673r- is there a typo here?

**Comment by jansunil** [comment]
@D-Disha  - I do see some Eval board links for the nanodac boards that do support the SDP-120 form factor.
Example: 
https://www.analog.com/en/resources/evaluation-hardware-and-software/evaluation-boards-kits/eval-ad5675.html#eb-overview
https://www.analog.com/en/resources/evaluation-hardware-and-software/evaluation-boards-kits/eval-ad5686r.html#eb-overview

Any reason why we haven't added them here?

**Review by jansunil — APPROVED** [review]

**Review by SaikiranGudla — APPROVED** [review]

**Pull request merged**